bvm/ballerina-runtime/src/test/java/io/ballerina/runtime/test/config/ConfigTest.java

```diff
@@ -64,8 +64,8 @@ public class ConfigTest {
 
     private static final Module ROOT_MODULE = new Module("rootOrg", "mod12", "1");
     private static final Type[] COLOR_ENUM_MEMBERS = new Type[]{
-            new BFiniteType("Colors", Set.of(StringUtils.fromString("RED")), 0),
-            new BFiniteType("Colors", Set.of(StringUtils.fromString("GREEN")), 0)};
+            new BFiniteType("COLOR_RED", Set.of(StringUtils.fromString("RED")), 0),
+            new BFiniteType("COLOR_GREEN", Set.of(StringUtils.fromString("GREEN")), 0)};
     public static final Type COLOR_ENUM_UNION = new BUnionType(COLOR_ENUM_MEMBERS, COLOR_ENUM_MEMBERS, 0, false,
             SymbolFlags.ENUM);
```

Comment:
Shall we have something like this?
```java
    private static final List<Type> COLOR_ENUM_MEMBERS = List.of(
            new BFiniteType("Colors", Set.of(StringUtils.fromString("RED")), 0),
            new BFiniteType("Colors", Set.of(StringUtils.fromString("GREEN")), 0));
    public static final Type COLOR_ENUM_UNION = new BUnionType(COLOR_ENUM_MEMBERS, "Colors", ROOT_MODULE, 0, false,
            SymbolFlags.ENUM);
```
This will be a similar scenario to the ballerina code.